### PR TITLE
Fission glossary

### PR DESCRIPTION
**Pull request by icidasset**
First attempt at writing the Fission glossary.
Let me know if I need to explain anything else.

### CODE REVIEW

File: fission/Glossary.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
+- **Webnative**: The name of the Fission SDK. Currently focused on Fission infrastructure, the goal is to be fully decentralised. [Learn more](https://guide.fission.codes/developers/webnative).
+- **WNFS**: Short for Web Native File System. A filesystem with both encrypted and public files. Access is granted through private keys. [Learn more](https://guide.fission.codes/developers/webnative/file-system-wnfs).
+- **Auth lobby**: The authentication/authorisation lobby. Here lives your private key that has access to everything in your filesystem. From here you can link other devices and grant apps to specific parts of your data. [Learn more](https://guide.fission.codes/developers/webnative/auth).
+- **DID**: Decentralised identifier, used to identify things or people. Fission accounts have both an associated username and a DID. [Learn more](https://en.wikipedia.org/wiki/Decentralized_identifiers).
+- **UCAN**: User Controlled Authorization Network, is a specialised JWT (JSON Web Token) that allows you to do decentralised authorisation (not needing a centralised server). The lobby issues UCANs to apps and your other devices. [Learn more](https://github.com/ucan-wg/ts-ucan).
```
Comment (NoelDeMartin) [inline fission/Glossary.md:7]: I don't know much about Fission, but this sounds confusing to me. Shouldn't it be called a "UCAN token" or something? I would say that the lobby issues tokens, but it's weird to say that the lobby issues "networks". Maybe thats how you call it in fission, though, no idea. Just asking :).

**Review by NoelDeMartin — COMMENTED** [review]
It looks good to me, but I'll wait for @rosano to comment because he knows more about Fission.

One thing that I also find interesting to mention, although it may not be relevant for this app, is the "Fission drive". I think that would be the equivalent to a Solid POD, right? That's where your data lives. Although I understand the data is actually stored in the IPFS network, so the drive is just a cache maintained by the Fission protocol in the server? (Or is it something specific to Fission the company, but not part of the protocol?).

On that note, what are the names of the company and the protocol? You call the protocol "Fission protocol" and the company behind it just "Fission"? Or that's not something you typically talk about, given that currently there are no other providers?

**Review by rosano — COMMENTED** [review]
@icidasset Great start. I think those are the main ones. There might be some concepts from the universes of IPFS (pinning, nodes, IPFS in the browser?) or WebCrypto (non exportable key?) that could be useful to include so that people know which aspects are present in Fission. But maybe that's redundant. Maybe worth asking the team if there are others or if there's something from these pages that belongs in a list for someone with no knowledge of Fission:
- https://guide.fission.codes/developers/concepts
- https://guide.fission.codes/accounts/account-signup

But I'm okay with merging this: we can always improve later. Thanks for taking the initiative.

**Comment by rosano** [comment]
@NoelDeMartin your questions might be a result of me explaining it poorly at the swap.

> "Fission drive". I think that would be the equivalent to a Solid POD, right? That's where your data lives. Although I understand the data is actually stored in the IPFS network, so the drive is just a cache maintained by the Fission protocol in the server? (Or is it something specific to Fission the company, but not part of the protocol?).

The Drive is actually just a file manager app that requests access to the entire filesystem: any app can do this in the same way that maybe a Solid app can request access to the entire POD. The data doesn't live there but either on your device or encrypted on IPFS.

> On that note, what are the names of the company and the protocol? You call the protocol "Fission protocol" and the company behind it just "Fission"? Or that's not something you typically talk about, given that currently there are no other providers?

I might be the only person that calls Fission a protocol. They're a company and they offer a 0data SDK, most or all of which is open-source. Perhaps it's not exactly a protocol like remoteStorage or Solid as it currently depends on Fission as the sole provider. But as mentioned in the glossary, the goal is to be more decentralized, or as @bmann might say, a 'constellation provider'.

**Comment by NoelDeMartin** [comment]
> your questions might be a result of me explaining it poorly at the swap

Well don't discard my lack of memory either :sweat_smile: But thanks for clarifying! Now that it's written down I can come back to read it again in the future :).

> The Drive is actually just a file manager app that requests access to the entire filesystem: any app can do this in the same way that maybe a Solid app can request access to the entire POD. The data doesn't live there but either on your device or encrypted on IPFS.

I see, in that case, is there a name for the copy that Fission keeps for you? Or is that just "Fission's server" and is not something that has a specific name? I understand that IPFS files are not stored in any place in particular, but I'm referring to the copy that Fission guarantees that won't be lost (as opposed to plain IPFS, where data can be lost if nobody else was sharing it and you lost your device).

**Comment by rosano** [comment]
> is there a name for the copy that Fission keeps for you? Or is that just "Fission's server" and is not something that has a specific name? I understand that IPFS files are not stored in any place in particular, but I'm referring to the copy that Fission guarantees that won't be lost (as opposed to plain IPFS, where data can be lost if nobody else was sharing it and you lost your device).

As I understand, there isn't a 'Fission server' and yes they make sure there is a copy available in case no other nodes can serve them. Someone correct me if needed but I think this is called 'pinning'; might be useful to have in the glossary.

**Comment by rosano** [comment]
@NoelDeMartin let's merge and improve later?

**Comment by NoelDeMartin** [comment]
@rosano ok, just did it. The only open thread I see is that doubt about UCAN vs UCAN token, but we can improve later.

**Pull request merged**